src/pages/index.tsx

```diff
@@ -16,8 +16,7 @@ const IndexPage: FunctionComponent = ({homePageData}: any) => {
         openGraph={{
           images: [
             {
-              url:
-                'https://res.cloudinary.com/dg3gyk0gu/image/upload/v1617697492/next.egghead.io/cards/egghead.io-digital-garden-cli-with-rust_2x.png',
+              url: 'https://res.cloudinary.com/dg3gyk0gu/image/upload/v1621135706/egghead-next-pages/dom-events/main-ogImage-dom-events_2x.png',
```

Comment:
Feature course in the index OgImage:

![image](https://user-images.githubusercontent.com/57044804/118385030-4a4f6e80-b5c0-11eb-8b7e-8d937ba93f19.png)